ci-operator/step-registry/upi/libvirt/infra/cleanup/pre/upi-libvirt-infra-cleanup-pre-commands.sh

```
@@ -0,0 +1,111 @@
+HOSTNAME="$(yq-v4 -oy ".\"${LEASED_RESOURCE}\".hostname" "${CLUSTER_PROFILE_DIR}/leases")"
+if [[ -z "${HOSTNAME}" ]]; then
+  echo "Couldn't retrieve hostname from lease config"
+  exit 1
+fi
+
+REMOTE_LIBVIRT_URI="qemu+tcp://${HOSTNAME}/system"
+VIRSH="mock-nss.sh virsh --connect ${REMOTE_LIBVIRT_URI}"
+echo "Using libvirt connection for $REMOTE_LIBVIRT_URI"
+
+# Test the remote connection
+mock-nss.sh virsh -c ${REMOTE_LIBVIRT_URI} list
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)